core/src/main/java/org/elasticsearch/common/geo/builders/PolygonBuilder.java

```diff
@@ -141,9 +142,10 @@ protected void validateHole(LineStringBuilder shell, LineStringBuilder hole) {
 
         Edge[] edges = new Edge[numEdges];
         Edge[] holeComponents = new Edge[holes.size()];
-        int offset = createEdges(0, orientation, shell, null, edges, 0);
+        final AtomicBoolean translate = new AtomicBoolean(false);
```

Comment:
I don't think this needs to be an AtomicBoolean? Its no longer a member variable so it should already be thread safe.